interfaces/semantic/types/array.type.js

```diff
@@ -11,14 +11,16 @@ describe('Semantic Interface', function() {
       ////////////////////////////////////////////////////
 
       it('should store proper array value', function(done) {
-        Semantic.User.create({ list: [0,1,2,3] }, function(err, record) {
+        Semantic.User.create({ list: [0,1,2,3] }, function(err, createdRecord) {
           assert(!err);
-          assert(Array.isArray(record.list));
-          assert(record.list.length === 4);
```

Comment:
We should still check if the `createdRecord` returns the right value, like: `assert(createdRecord.list.length === 4);` for all types.